src/app/cases/case_components/case-contacts/case-contacts.component.ts

```diff
@@ -36,4 +73,50 @@ export class CaseContactsComponent {
     this.defaultColumns = defaultColumnDefs;
     this.resourceService = resourceService;
   }
+
+  exportCustomCase(): void {
+    this.dialog.open(CustomExportComponent, {
+      width: EXPORT_CUSTOM_MODAL_WIDTH,
+      data: {
+        exportType: EXPORT_TYPE.CASE,
+        exportFormData: FORM_DATA_EXPORT_CONFIGURATION,
+      },
+    });
+  }
+
+  exportBasicCase(): void {
+    this.notificationService.prompt({
+      title: this.translateService.instant('captions.exportBasic'),
+      message: this.translateService.instant('strings.infoDownloadExport'),
+      maxWidth: SMALL_NOTIFICATION_MODAL_WIDTH,
+    });
+
+    this.exportService.executeExport(EXPORT_TYPES.BASIC, API_ROUTE_CASES.EXPORT);
```

Comment:
We're no the cases contact tab, so I expect this to be an export of contacts. This should be API_ROUTE_CONTACTS.